runtime/src/mm/cpp/CurrentException.hpp

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2010-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+
+#ifndef RUNTIME_MM_CURRENT_EXCEPTION_H
+#define RUNTIME_MM_CURRENT_EXCEPTION_H
+
+#include "Types.h"
+#include "Utils.hpp"
+
+struct ObjHeader;
+
+namespace kotlin {
+namespace mm {
+
+class CurrentException : private Pinned {
```

Comment:
(minor) May be rename to "CurrentException**s**", since we allow adding several of them?